requirements.txt

```diff
@@ -6,7 +6,7 @@ chardet==3.0.4
 click==7.1.2
 cycler==0.10.0
 cymem==2.0.4
-en-core-web-sm==2.2.5
+
```

Comment:
Please add requirement back into file

Reply:
😳 How?

Reply:
Hmmm actually come to think of it, this didn't actually work when you tried to inatt it like this, so maybe let's leave it out of the requirements file and I'll find a better solution. So ignore this comment.